Assets/Scripts/ZoomSystem/Scanner.cs

```diff
@@ -85,19 +95,32 @@ public void Zoom(InputAction.CallbackContext ctx)
 
     public void Scan(InputAction.CallbackContext ctx)
     {
-        //Debug.Log("Scan attempted.");
+        UpdateAreaInView();
+        ScanPerformed?.Invoke(areaInView);
 
-        LensSettings lens = cam.m_Lens;
-        float viewHeight = 2f * lens.OrthographicSize;
-        float viewWidth = viewHeight * lens.Aspect;
-        areaInView.Set(
-            cam.transform.position.x - (viewWidth / 2),
-            cam.transform.position.y - (viewHeight / 2),
-            viewWidth, viewHeight);
+    }
 
-        //Debug.Log($"Area in view determined: {areaInView}");
+    public void Enhance(InputAction.CallbackContext ctx)
+    {
+        /*if (zoomLevel < maxZoomLevel)
+        {
+            Debug.Log("An enhance can only be performed at max zoom level.");
```

Comment:
Should we fire an event when this happens so that we can show a message to the player to zoom more?

Reply:
Yeah, good idea. I actually commented this out because for some reason I couldn't get zoomLevel == maxZoomLevel, and just by playing it was actually mildly annoying to have to remember - in theory the hotspots checking for overlap % with the camera view already force the player to zoom close enough. But I'll put in some scaffolding for this anyway!